### ISSUE TITLE
Compatibility with Enhanced PlayerModel Selector

### ISSUE THREAD

**Issue by Alf21** [TTT-2/TTT2#226]
https://steamcommunity.com/sharedfiles/filedetails/?id=504945881&searchtext=Enhanced+PlayerModel+Selector

**Comment by Deathbyfarting** [comment]
Just to add my two sense here. I've been trying things and it looks like you can use these two mods together however the ttt2 forces the player model right before starting the round. To have the two work together simply disabling "force player model" for both client and server works (though defeats the purpose)

I'm not a full programmer and don't know the mod code but it seems like the model change is a parent of the round starting. To bring Compatability you just need to change the model force to a boolean and make it independent of the round starting. Again not a full programmer but it seems simple from the little I know of the art.

**Comment by Alf21** [comment]
That's a way more complicated :/ Because every model add-on handles setting the model in another way (timers, direct access, hook access, caching).
I've focussed on PS and TTT2 add-on compatibility. Maybe there is another way, but there isn't a simple solution.

**Comment by Deathbyfarting** [comment]
The reason why I brought up Enhanced PlayerModel selector is that it's the best I could find (I've tried most in search of a fix and there mostly garbage impersonations). In my search I found others and they all pretty much "turned on", but not easily or actually changed the model.

I watch a lot of AH (Achievement Hunter) play TTT and play some myself so the ideas and concepts that come from everyone looking the same isn't lost on me. However, I've also seen Yogscast a couple of times who play a HEAVILY modded version and really enjoy being able to tell whos in a room at a glance because of their models.

Overall I get it's complicated and probably really hard to run down. My main point here is some people want custom models and others may not. Being able to say you're compatible with literally the most feature-rich and easy to use playermodel changer **_OR_** implementing similar features into TTT2 natively my help some groups trans from the original mod.

Just sayin'...

**Comment by TimGoll** [comment]
The thing is, we really want to support all these model mods. But it is tricky

**Comment by GTREDITION** [comment]
please fix the compatibility :(

**Comment by Fakas** [comment]
[I've submitted a PR to fix this particular compatibility issue.](https://github.com/TTT-2/TTT2/pull/414)

**Comment by Histalek** [comment]
As this was presumably fixed and merged in #414 (thank you very much @Fakas ) i will close this issue.

If there are further problems feel free to open up a new issue :)
